features/step_definitions/cluster_configuration/localstorage.rb

```diff
@@ -1,245 +1,95 @@
-  serviceaccount="local-storage-admin"
-  configmap="local-volume-config"
-  template="local-storage-provisioner"
-  image="#{img_registry}/openshift3/local-storage-provisioner:#{img_version}"
-  path ||="/mnt/local-storage"
-  cmurl = "#{BushSlicer::HOME}/testdata/storage/localvolume/configmap-37.yaml"
-  if env.version_ge("3.10", user: user)
-    cmurl = "#{BushSlicer::HOME}/testdata/storage/localvolume/configmap.yaml"
-  end
-
-  project(namespace)
-  # if project exists, delete project and pvs created by local storage provisioner
-  if project.exists?(user: admin)
-    project.delete(by: admin)
-    project.wait_to_disappear(admin)
-
-    BushSlicer::PersistentVolume.list(user: admin).each { |pv|
-      pv.delete(by: admin) if pv.name.start_with?("local-pv-") && (pv.local_path&.start_with?("#{path}/fast") || pv.local_path&.start_with?("#{path}/slow"))
-    }
+  namespace = "openshift-local-storage"
+  name = "local-storage"
+
+  # create namespace
+  unless project(namespace).exists?
+    step %Q/I run the :oadm_new_project admin command with:/, table(%{
+      | project_name  | #{namespace} |
+    })
+    step %Q/the step should succeed/
   end
 
-  ns_cmd = "oc adm new-project #{namespace} --node-selector=''"
-  @result = env.master_hosts.first.exec(ns_cmd)
-  step %Q/the step should succeed/
-
-  env.hosts.each do |host|
-    setup_commands = [
-      "mkdir -p #{path}/fast/vol1",
-      "mount -t tmpfs fvol1 #{path}/fast/vol1",
-      "mkdir -p #{path}/fast/vol2",
-      "mount -t tmpfs fvol2 #{path}/fast/vol2",
-      "mkdir -p #{path}/slow/vol1",
-      "mount -t tmpfs svol1 #{path}/slow/vol1",
-      "mkdir -p #{path}/slow/vol2",
-      "mount -t tmpfs svol2 #{path}/slow/vol2",
-      "chcon -R unconfined_u:object_r:svirt_sandbox_file_t:s0 #{path}"
-    ]
-    res = host.exec_admin(*setup_commands)
-    raise "error preaparing subdirs for local storage provisioner" unless res[:success]
+  # create operator group
+  step %Q/I use the "#{namespace}" project/
+  unless operator_group(name).exists?
+    step %Q/I obtain test data file "storage\/localvolume\/operatorgroup.yaml"/
+    step %Q/I run oc create over "operatorgroup.yaml" replacing paths:/, table(%{
+      | ["metadata"]["name"]           | #{name}      |
+      | ["metadata"]["namespace"]      | #{namespace} |
+      | ["spec"]["targetNamespaces"][0]| #{namespace} |
+    })
+    raise "Create OperatorGroup #{name} failed" unless @result[:success]
   end
 
-  step %Q/I download a file from "#{cmurl}"/
-  cfm = YAML.load(@result[:response])
-  filepath = @result[:abs_path]
-  if env.version_ge("3.10", user: user)
-    cfm["data"]["storageClassMap"].gsub!("/mnt/local-storage", path)
-  else
-    cfm["data"]["local-fast"].gsub!("/mnt/local-storage", path)
-    cfm["data"]["local-slow"].gsub!("/mnt/local-storage", path)
+  # get channel
+  step %Q/evaluation of `cluster_version('version').channel.split('-')[1]` is stored in the :channel clipboard/
```

Comment:
@qinpingli when it's inside a step defintion, you don't need to have the cucumber step overhead.  Probably just `cb.channel = cluster_version('version').channel.split('-')[1]` would be enough.